igvc:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
    # max/min: 0.785
  left_steering_position_controller:
    type: effort_controllers/JointPositionController
    joint: left_steering_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  right_steering_position_controller:
    type: effort_controllers/JointPositionController
    joint: right_steering_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}


  # Effort Controllers -----------------------------------------
  left_rear_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: left_rear_axle
    pid: {p: 80.0, i: 0.00, d: 0.0}
  right_rear_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: right_rear_axle
    pid: {p: 80.0, i: 0.0, d: 0.0}